### ISSUE TITLE
Support WSTRING in structures.

### ISSUE THREAD

**Issue by stlehmann** [stlehmann/pyads#291]
WSTRINGS are supported since version 3.3.9. Structure handling should also support this. This should mainly affect the functions dict_from_bytes and bytes_from_dict. Also some testing needs to be added.

**Comment by stlehmann** [comment]
```python
if plc_datatype == PLCTYPE_STRING:
                    if str_len is None:
                        str_len = PLC_DEFAULT_STRING_SIZE
                    var_array.append(
                        bytearray(byte_list[index: (index + (str_len + 1))])
                        .partition(b"\0")[0]
                        .decode("utf-8")
                    )
                    index += str_len + 1
if plc_datatype == PLCTYPE_WSTRING:
    .... 
```

**Comment by stlehmann** [comment]
Solved with #293
